effects.cwt

```diff
@@ -1352,13 +1353,15 @@ alias[effect:approve_law] = law/title
 ###
 alias[effect:conquest_culture] = culture/character
 
-## scope = character/title
+# TODO confirm scope
+## scope = <province>
 ### Creates a hospital in the specified province
-alias[effect:create_hospital] = province (ID or scope pointing to province)
+alias[effect:create_hospital] = province
```

Comment:
```suggestion
alias[effect:create_hospital] = province
alias[effect:create_hospital] = scope[province]
```
If by this it means the RHS either needs to be a province id, or a province event_target, then this is probably what you want :)
`<province>` says a value of type province.
`scope[province]` says something that resolves to a scope object in scope province

Reply:
Actually, just alias[effect:create_hospital] = scope[province] will be enough, because <province> already can resolve to a province scope.